crates/ruma-events/src/room/redaction.rs

```diff
@@ -59,17 +56,9 @@ pub struct SyncRedactionEvent {
 
 /// A redaction of an event.
 #[derive(Clone, Debug, Deserialize, Serialize, EventContent)]
-#[ruma_event(type = "m.room.redaction")]
+#[ruma_event(type = "m.room.redaction", kind = Message)]
```

Comment:
I'm not so sure about this. Could you leave this file as-is for now? Or is this change important for other things to work?

Reply:
Well, the problem is that without it there is no `RedactedRedactionEventContent` since we are now only generating redacted structs for state and message events / event contents. We could special case it but that always blows up the complexity in the macro.

If I add this back `impl RedactedStateEventContent for RedactedRedactionEventContent {}` then the generated code is exactly the same as non generated marker trait wise. Are there other concerns or is this just a fuzzy area of redaction / the spec?

Reply:
This is kind of a fuzzy area of redaction, yes. I think I might change things up again, but I'm okay merging as-is.